Overview.bs

```diff
@@ -1440,40 +1447,45 @@ Some example inputs and the corresponding expansions:
   <tr>
     <td>//foo.bar/{id}</td>
     <td>123</td>
-    <td>//foo.bar/7B</td>
+    <td>//foo.bar/FC</td>
   </tr>
   <tr>
     <td>//foo.bar{/d1,d2,id}</td>
     <td>123</td>
-    <td>//foo.bar/B/7/7B</td>
+    <td>//foo.bar/C/F/FC</td>
   </tr>
   <tr>
     <td>//foo.bar{/d1,d2,d3,id}</td>
     <td>123</td>
-    <td>//foo.bar/B/7/0/7B</td>
+    <td>//foo.bar/C/F/_/FC</td>
   </tr>
 n    <tr>
     <td>//foo.bar{/d1,d2,d3,id}</td>
     <td>baz</td>
-    <td>//foo.bar/z/a/b/baz</td>
+    <td>//foo.bar/K/N/G/C9GNK</td>
   </tr>
   </tr>
     <tr>
     <td>//foo.bar{/d1,d2,d3,id}</td>
-    <td>az</td>
-    <td>//foo.bar/z/a/0/az</td>
+    <td>z</td>
+    <td>//foo.bar/8/F/_/F8</td>
   </tr>
   </tr>
     <tr>
     <td>//foo.bar{/d1,d2,d3,id}</td>
     <td>àbc</td>
-    <td>//foo.bar/c/b/%C3%A0/%C3%A0bc</td>
+    <td>//foo.bar/0/0/4/OEG64OO</td>
+  </tr>
+  </tr>
+    <tr>
+    <td>//foo.bar{id64}</td>
+    <td>àbc</td>
+    <td>//foo.bar/w6BiYw==</td>
```

Comment:
In this example the ='s should be percent encoded. You could add an additional example that uses {+id64} which would leave the ='s untouched.